### DISCUSSION TITLE
Text colours not working correctly on VS Code built in terminal

### DISCUSSION THREAD

**Discussion by Ant2210** [JanDeDobbeleer/oh-my-posh#5274] [Troubleshoot]
Text colours don't seem to be working properly in vs code, is there a setting I could have missed in vs code?

### Terminal
![image](https://github.com/user-attachments/assets/c50ed7c3-0bc1-4a00-a5b5-df5c9b6cb93a)

### VS Code
![image](https://github.com/user-attachments/assets/7e6f8f81-8db0-449c-8f23-3349e918090e)

I am also having an issue on first load in VS Code, it doesn't seem to render background colours correctly until I start another new line.

### First Load
![image](https://github.com/user-attachments/assets/773eeb64-3620-46e4-9ee7-93ef6cf16d5e)

### After new line started
![image](https://github.com/user-attachments/assets/6ad122cd-8fa1-4398-9c15-09fd03a1de5f)

Thanks in advance. 😊

**Comment by JanDeDobbeleer** [comment]
Both issues are for vscode. Not sure why, they're not coloring correctly. Already seen both reported before, and oh-my-posh has no different output in both.

**Reply by Ant2210** [reply]
Thank you anyway.
